go.mod

```
@@ -1,6 +1,6 @@
 module github.com/Scalingo/go-philae/v5
 
-go 1.24.3
+go 1.24.0
```

We don't want to specify the patch version so that it uses the latest patch version for 1.24.

```suggestion
go 1.24
```

---

As stated in the PR comment:

> We cannot set it to 1.24 because [this](https://github.com/fsouza/go-dockerclient) dependency is explicitly setting to 1.24.0.

I don't think this is possible, or I don't know how to do it.

---

You modify the go.mod content, commit and push it? :thinking: 

At least that's what I did everywhere

---

No you need to run `go mod tidy` to ensure the file is correct. And if I run it on the repo, it bumps back to `1.24.0`. At least that's what I did :smile: .